spec/unit/bricks/middleware/mask_logger_decorator_spec.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'spec_helper'
+
+require 'gooddata/bricks/middleware/mask_logger_decorator'
+
+describe GoodData::Bricks::MaskLoggerDecorator do
+  let(:logger) { double(Logger) }
+  let(:values_to_mask) {%w(secret_password sensitive)}
```

Comment:
warning: Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`. (rubocop)